lib/ores_api.rb

```diff
@@ -57,8 +68,15 @@ def query_url(rev_ids)
   end
 
   def ores_server
-    conn = Faraday.new(url: 'https://ores.wikimedia.org')
+    conn = Faraday.new(url: ORES_SERVER_URL)
     conn.headers['User-Agent'] = ENV['dashboard_url'] + ' ' + Rails.env
     conn
   end
+
+  def invoke_error_handling(error, url, response_body)
```

Comment:
This method is still very similar across APIs. I think it could be done such that a single method (maybe rename `log_error` to `handle_api_error`) with optional keyword arguments is used across all these APIs, and none of them define their own `invoke_error_handling` method, they just call the one provided by the module with whatever arguments are relevant for this particular API.

Reply:
The reasons why I have made this is for passing of arguments. `sentry_extra` is different for all the three APIs, and also `invoke_error_handling` calls in Replica pass different arguments in 2 places. I think moving the method would have the same effect, and there could be problems with passing of the high number of optional arguments

Reply:
Or I could do it similar to how`UpdateLogger.update_course` is called? (Talking about the way `times` argument is passed there)

Reply:
Yes, if all the extra optional data is an arbitrary hash of stuff to pass in to sentry, collecting it all up as an implicit single argument hash (like with `UpdateLogger.update_course`) makes perfect sense.